newIDE/app/src/StringSelectorEditor/index.js

```diff
@@ -0,0 +1,66 @@
+// @flow
```

Comment:
Let's rename this to StringArrayEditor? Also I wonder if this can be in Utils/ as it's fairly generic.

(Selector was too linked to its previous usage)

Reply:
Renaming it is a good idea :+1:

For the location of the file, If I follow the current logic, `ObjectTypeSelector` and `BehaviorTypeSelector` live in `/src` - whereas `SelectField`, `SemiControlledTextField` live in `/src/UI`
I'm tempted to move it to `src/UI` too (next to `LocalFilePicker`, `ColorField`...)

Reply:
As discussed, leaving it in `/src` as this is a component and not really a util, and it's quite specific to GDevelop, so cannot be fully considered as a "generic" UI component